lxd/instance/drivers/driver_lxc.go

```diff
@@ -6567,6 +6568,40 @@ func (d *lxc) inheritInitPidFd() (int, *os.File) {
 	return -1, nil
 }
 
+type sftpProcess struct {
+	runningForkfile string
+	conn            net.Conn
+	cancel          *cancel.Canceller
+}
+
+var sftpMu sync.Mutex
+var sftpProcesses = make(map[string]*sftpProcess)
+
+// stop sends a cancel signal to the sftp service.
+// If ctx doesn't have a deadline then a default timeout of 5s is added.
+func (p *sftpProcess) stop(ctx context.Context) error {
```

Comment:
This function doesn't appear to "sends a cancel signal to the sftp service."

Reply:
fixing that